odex2apk.py

```diff
@@ -38,7 +38,7 @@
 OAT2DEX = os.getenv("OAT2DEX", os.path.join(_dirname, "oat2dex.jar"))
 
 # Supported architectures (first match will be used)
-architectures = ["arm", "arm64", "x86", "x86_64"]
+architectures = ["x86_64", "x86", "arm64", "arm"]
```

Comment:
This changes the detection order, so if both `arm64` and `arm` exists, then the former will now be used. Is this really the right logic? Would it be more accurate if the `detect_arch` function is modified to check for presence of another file?